src/test/s3/presignedUrlWizard.test.ts

```diff
@@ -0,0 +1,55 @@
+/*!
+ * Copyright 2023 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { createWizardTester, WizardTester } from '../shared/wizards/wizardTestUtils'
+import { PresignedUrlWizard, PresignedUrlWizardState } from '../../s3/commands/presignedURL'
+import { S3FileNode } from '../../s3/explorer/s3FileNode'
+
+describe('PresignedUrlWizard', function () {
+    let tester: WizardTester<PresignedUrlWizardState>
+
+    describe('invoked from command palette', function () {
```

Comment:
is this describe() needed (or accurate)? some of the it() tests are "from node" instead of "from command palette". Looks like we don't need this extra describe() scope

Reply:
Unintentionally left this in, it's been removed.